app/code/Magento/Sales/Ui/Component/Listing/Column/PurchasedPrice.php

```diff
@@ -51,14 +53,9 @@ public function prepareDataSource(array $dataSource)
         if (isset($dataSource['data']['items'])) {
             foreach ($dataSource['data']['items'] as & $item) {
                 $currencyCode = isset($item['order_currency_code']) ? $item['order_currency_code'] : null;
-                $item[$this->getData('name')] =
-                    $this->priceFormatter->format(
-                        $item[$this->getData('name')],
-                        false,
-                        null,
-                        null,
-                        $currencyCode
-                    );
+                $purchaseCurrency = $this->currencyFactory->create()->load($currencyCode);
```

Comment:
Refer to my comment above related to the `load()` method.

Reply:
![image](https://user-images.githubusercontent.com/8234209/62197175-96103400-b3a9-11e9-866f-f6e85c0e5cfc.png)

It is not deprecated. Please check it.